### PR TITLE
Global state spring clean

### PR DESCRIPTION
**Pull request by UncleClapton**
Cleaning global state is fun \o/

### CODE REVIEW

File: pages/my/characters.js [hunk 1]
```diff
@@ -52,7 +56,7 @@ class MyCharacters extends Component {
 
         {!loading && (
           <ul className="characters">
-            {this.props.characters.map(character => (
+            {Object.values(characters || {}).map(character => (
```
Comment (trezy) [inline pages/my/characters.js]: Ho, shiiiiiiiiiiiiiiiit... I've never used `Object.values()`. It makes so much sense! 🎊🎉

File: store/reducers/authentication.js [hunk 2]
```diff
@@ -15,11 +15,9 @@ export default function (state = initialState.authentication, action) {
...
     case actionTypes.CONFIRM_ACCOUNT:
     case actionTypes.LOGIN:
-      if (typeof status !== 'undefined') {
-        return {
-          ...state,
-          loggedIn: status === 'success',
-        }
+      return {
+        ...state,
+        loggedIn: typeof status === 'string' && status === 'success',
```
Comment (trezy) [inline store/reducers/authentication.js]: Mmmmmmmmm... That's a tasty one-liner. 😋

File: pages/my/characters.js (hunk 1 above)
Comment (UncleClapton) [reply pages/my/characters.js]: The Object class has a lot of nifty things in it. 👍

File: store/reducers/authentication.js (hunk 2 above)
Comment (UncleClapton) [reply store/reducers/authentication.js]: a bit more simple, yes :-)

**Pull request merged**